### PR TITLE
CSS fixes for headings

### PR DESCRIPTION
**Pull request by gravitystorm**
This fixes a number of issues with the headings, by removing some reset rules (that are acting after the bootstrap reset + defaults and so are messing things up) and by removing some custom heading font sizes and allowing the bootstrap values to be used.

I've overridden the heading sizes for the sidebar temporarily (roughly making h2 the same as bootstrap h3, and h3/h4 the same as bootstrap h5) since there's a bunch of specific style rules preventing me from changing e.g. h2 -> h3. For example, the h2 in sidebar gets extra left padding, since it isn't included in the sidebar browse-section. Fixing all that will come in a future PR.

### CODE REVIEW

**Comment by tomhughes** [comment]
I've merged this for now but it's definitely going to need some more work as I think things kind of swings things back the other way - the h3/h4 items in the side bar are increased from 13px to 20px in size which makes them nearly 50% larger than before.

File: app/assets/stylesheets/common.scss [hunk 1]
```diff
@@ -542,8 +541,6 @@ body.compact {
   #sidebar {
     float: left;
     width: $sidebarWidth;
-    background: #fff;
```
Comment (tomhughes) [inline app/assets/stylesheets/common.scss]: Removing the background colour here means the welcome message for new users has no background and the text is overlaid directly on the map.

Other uses of the sidebar seem to work somehow, despite not having an explicit backgound colour - they seem to inherit the browser default I think but that doesn't happen with the welcome message?

Comment (gravitystorm) [reply app/assets/stylesheets/common.scss]: OK, I'll investigate and fix this.

**Comment by woodpeck** [comment]
FWIW, the new headings appear quite big to me too; any heading like "Node: Somename (12345)" will now wrap on my screen, and any changeset comment with more than 6 words will wrap too. Useful changeset comments can easily span four lines; if you look at someone's edit history, the sidebar is practically one single rather-big-font headline. I guess one can get used to it but it definitely takes some getting used to...

**Comment by HolgerJeromin** [comment]
Ref #2492

**Comment by enockseth** [comment]
Agree with @woodpeck.  Font sizes way too big for me too.

**Pull request merged**